.github/workflows/runtests.yml

```diff
@@ -23,16 +23,13 @@ jobs:
           python-version: '3.11'
       - run: python -m pip install --upgrade pip
 
-      - run: pip install mpmath flake8 flake8-comprehensions ruff pytest
+      - run: pip install mpmath pytest pre-commit
 
       - name: Basic code quality tests
         run: bin/test quality
 
-      - name: Run flake8 on the sympy package
-        run: flake8 sympy
-
-      - name: Run Ruff on the sympy package
-        run: ruff check .
+      - name: Run flake8 and ruff on the sympy package with pre-commit
+        run: pre-commit install; pre-commit run -a
```

Comment:
Is the install step actually needed here?

Reply:
Yes, you need to install pre-commit in the current repository before you can run it.

Reply:
Are you sure about that?

To be clear if this change is merged I would like to be able to run pre-commit but I would not want to install the git hook.

Reply:
Sorry, I think you're right. You can still run it when it's not installed.